package.json

```diff
@@ -40,7 +39,7 @@
     "jsdom": "3.0.0",
     "jshint": "^2.6.3",
     "react": "^0.14.0",
-    "react-router": "^1.0.3",
+    "react-router": "^2.4.0",
```

Comment:
@samsel the react router dependency update broke our app since our React Router code, with a package.json dependency of ~1.0.3, does not work with React Router 2.x.  This package.json dependency chain is interesting since React Engine went from working, with a package.json dependency of ^3.3.0, to not working when React Engine update to 3.4.  We run npm prune and then npm install, but the issue was not revealed until our build did a clean install of the node packages i.e. you may not experience this issue until node_modules is removed and re-installed.  Maybe you want to consider a major version bump with this type of update?

The fix for this issue was to freeze React Router at 1.0.3 in our package.json.